turingschool-examples/whats-cookin-starter-kit
# Feature/single recipe view

## Is this a fix or a feature?
- feature

## What is the change?
- This adds the HTML and CSS for a "single recipe view" 
- User Story: when a user clicks a recipe, they should be able to see recipe information such as: 
  - ingredients
  - instructions
- Additionally from this view the user can add to their favorites, and add to a cook list
- There are navigational buttons for the user to return to the recipe list, the favorites section, or their pantry

## What does it fix?
- N/A

## Where should the reviewer start?
- index.html: lines: 145- 201
- style.css: lines: 187-283

## How should this be tested?
-open index.html and match single view layout to wireframe and user story/UX
- test layout on browser screen and phone screen, layout should be responsive

## Screenshots (if appropriate):
